locales/ru_RU/LC_MESSAGES/duckduckgo.po

```diff
@@ -7453,7 +7453,7 @@ msgid ""
 "Viewing ads is privacy protected by DuckDuckGo. Ad clicks are managed by "
 "Microsoft's ad network"
 msgstr "DuckDuckGo обеспечивает защиту конфиденциальности пользователей при просмотре рекламных объявлений. Управление переходами по рекламным объявлениям осуществляется"
-" рекламной сетью «Майкрософт»"
+" рекламной сетью Майкрософт"
```

Comment:
In order to merge this, I need you to please also update the corresponding string in https://github.com/ViktorOn/duckduckgo-locales/blob/c1dafd342036217049ece8ca14e925b1ab3af012/t/internal_translations.t#L127. Thank you!

Reply:
It's done